platform/linux-generic/include/odp_packet_internal.h

```diff
@@ -107,9 +107,6 @@ typedef struct ODP_ALIGNED_CACHE odp_packet_hdr_t {
 
 	uint16_t tailroom;
 
-	/* Event subtype */
-	int8_t   subtype;
-
```

Comment:
This gap could now be used to store for example `cos` field, which is commonly used in the fast path.